packages/superset-ui-core/src/chart/components/NoResultsComponent.tsx

```diff
@@ -39,11 +40,11 @@ const NoResultsComponent = ({ className, height, id, width }: Props) => {
       className={className}
       id={id}
       style={containerStyles}
-      title={shouldRenderBody ? undefined : BODY_STRING}
+      title={shouldRenderBody ? undefined : t(BODY_STRING)}
     >
       <div style={MESSAGE_STYLES}>
-        <div style={TITLE_STYLES}>No Results</div>
-        {shouldRenderBody && <div style={BODY_STYLES}>{BODY_STRING}</div>}
+        <div style={TITLE_STYLES}>{t('No Results')}</div>
+        {shouldRenderBody && <div style={BODY_STYLES}>{t(BODY_STRING)}</div>}
```

Comment:
I think it would be better to add the translation to the actual constant value on line 8, eg
```js
const BODY_STRING =
  t('No results were returned for this query. If you expected results to be returned, ensure any filters are configured properly and the datasource contains data for the selected time range.');
```

Reply:
This will not pass CI...

Reply:
what's the error

Reply:
@nytai https://github.com/apache-superset/superset-ui/runs/2671917720

![image](https://user-images.githubusercontent.com/18044730/121458256-9f25af80-c9db-11eb-87f5-2df00e8fe620.png)

Reply:
You can also move `const BODY_STRING = t(...)` into `NoResultsComponent`. In general, we want translation strings to use literal keys instead of variables (it makes future refactoring easier).

Reply:
@ktmud Done!